nncf/experimental/common/pruning/operations.py

```diff
@@ -173,29 +175,16 @@ def _both_dim_blocks_exist(left_idx, right_idx):
                 right = right_dim_groups[input_shape_len - 2]
                 assert len(left) == 1 and len(right) == 1, "multiple groups are not supported"
                 group = PropagationGroup.join_groups(left[0], right[0])
-                cls.add_consumer_block(group, node)
+                pruning_dimension = 1
+                if node.layer_attributes is None:
+                    pruning_dimension = None
+                consumer = ConsumerInfo(node_id=node.node_id, pruning_dimension=pruning_dimension)
```

Comment:
I think current node is not a consumer in a sense of a node that have a weight that should be adjusted (for example, consumer conv input channels should be adjusted to make pruning possible), so pruning dimension always should be None?

Reply:
you are right, corrected